### DISCUSSION TITLE
Modules created with `blob` and `createObjectURL` fail to load

### DISCUSSION THREAD

**Discussion by JayaKrishnaNamburu** [webpack/webpack#14870] [Q&A] [answered]
I am trying to load a module that is created using `blob` and `createObjectURL` and it keeps throwing Unable to resolve module.

Here is the piece of code

```js
(async () => {
  const code = `console.log('Testing some logs')`;
  const blob = new Blob([code], { type: 'text/javascript' });
  const url = URL.createObjectURL(blob);
  await import(url)
})()
```

And here is repro URL https://stackblitz.com/edit/github-ljdydt?file=src%2Findex.js

### Errror
```
Uncaught (in promise) Error: Cannot find module 'blob:https://github-ljdydt--5000.local.webcontainer.io/70579cfc-9652-48c1-80dd-92f2a99a8b57'
```

<img width="1396" alt="Screenshot 2021-11-30 at 12 41 16 PM" src="https://user-images.githubusercontent.com/11075561/144002718-77481f1e-d869-4b67-a396-ee1c3b6b0ec1.png">

**Comment by alexander-akait** [comment, marked as answer]
Please use `await import(/* webpackIgnore: true */ url)` to say webpack ignore this import

**Reply by JayaKrishnaNamburu** [reply]
As helpful as always with webpack related doubts @alexander-akait . Thank you so much :)
